src/equations/shallow_water_1d.jl

```diff
@@ -582,9 +658,14 @@ end
 
 # Calculate the error for the "lake-at-rest" test case where H = h+b should
 # be a constant value over time
-@inline function lake_at_rest_error(u, equations::ShallowWaterEquations1D)
+@inline function lake_at_rest_error(u, u_exact, equations::ShallowWaterEquations1D)
```

Comment:
Does this alternative strategy for computing the lake-at-rest error still work for the old (completely wet) shallow water tests?

I am curious because the well-balanced test for the discontinuous bottom topography, e.g., `examples/tree_1d_dgsem/elixir_shallow_water_well_balanced.jl` has two versions of the initial conditions and I suspect this will be wrong for those cases.